install.sh

```diff
@@ -77,7 +77,9 @@ if [ $upgrade -eq 1 -a $makerfaire2018 -eq 0 -a -d ${home_dir}/wm8960 ]; then
   cd ${home_dir}/wm8960
   sudo chown -R ${owner} .git
   pull=`git pull`
-  if [ "$pull" != "Already up to date." ]; then
+  if [ "$pull" == "Already up to date." -o "$pull" == "Déjà à jour." ]; then
```

Comment:
This is fine for French, but what about other non-English languages?...
I think a simpler/more general solution would be to just add the line:
`export LANGUAGE=`
at the beginning of the script, so that `git` always outputs messages in the default (English) language.